### DISCUSSION TITLE
Dir ( ltr, rtl )

### DISCUSSION THREAD

**Discussion by aalhommada** [tailwindlabs/tailwindcss#10690] [Ideas]
I have a blog with two languages , english ( ltr ) and Arabic ( rtl ) .
when useing prose with dir = "rtl" , still get the code **dir**   rtl , which is wrong , 
``` <article className="prose prose-neutral" dir="rtl">```

![tailwind-typography](https://user-images.githubusercontent.com/48802189/221433115-5cc4cce2-e8fe-4cd7-82f5-5dc0ffbb888e.png)

Even when try ``` <article className="prose prose-code:text-left">```  it give the formate :

![tailwind-typography-align-code-left](https://user-images.githubusercontent.com/48802189/221433330-d1e9ffbc-1f9a-4c90-a8f3-2d1c9e4215d2.png)

Is there any suggestion to fix this ?

**Comment by wongjn** [comment]
You could try adding `dir="ltr"` to the `<pre>` element that wraps the code text. [Example Tailwind Play](https://play.tailwindcss.com/NtNKGXpo7e)

**Reply by aalhommada** [reply]
Thanks @wongjn for your answer 
but I am getting the content ( data ) from markdown file which come from headless cms, so it is not possible to separate the part that only contain the code from the other parts ( headin, p, image ......)

**Reply by wongjn** [reply]
You could apply `direction: ltr` to `<pre>` elements via `prose-pre:[direction:ltr]`, [example Tailwind Play](https://play.tailwindcss.com/ImOMydig8N).

**Reply by aalhommada** [reply]
Thanks a lot @wongjn , that is very helpful 👍
